2025-06-14 17:20:05,080 INFO: DEBUG_TAG: La valeur de RADARR_TAG_ON_ARCHIVE est 'os.environ.get('RADARR_TAG_ON_ARCHIVE', 'vu')' [in X:\MediaManagerSuite\app\plex_editor\routes.py:336]
2025-06-14 17:20:06,850 WARNING: Erreur 404 - Page non trouvée: http://localhost:5001/.well-known/appspecific/com.chrome.devtools.json (Référent: None) [in X:\MediaManagerSuite\app\__init__.py:79]
2025-06-14 17:20:16,771 INFO: Filtre date API: ajout_recent_jours = 100d [in X:\MediaManagerSuite\app\plex_editor\routes.py:180]
2025-06-14 17:20:16,773 INFO: show_library: Arguments finaux pour API search(): {'unwatched': False, 'addedAt>>=': '100d', 'sort': 'addedAt:desc'} [in X:\MediaManagerSuite\app\plex_editor\routes.py:223]
2025-06-14 17:20:17,137 INFO: Emprunt d'identité réussi pour 'User'. [in X:\MediaManagerSuite\app\plex_editor\routes.py:245]
2025-06-14 17:20:17,144 INFO: Exécution de library.search sur 'Films' pour utilisateur 'User' avec args: {'unwatched': False, 'addedAt>>=': '100d', 'sort': 'addedAt:desc'} [in X:\MediaManagerSuite\app\plex_editor\routes.py:270]
2025-06-14 17:20:17,163 INFO: DEBUG_TAG: La valeur de RADARR_TAG_ON_ARCHIVE est 'os.environ.get('RADARR_TAG_ON_ARCHIVE', 'vu')' [in X:\MediaManagerSuite\app\plex_editor\routes.py:336]
2025-06-14 17:20:17,511 WARNING: Erreur 404 - Page non trouvée: http://localhost:5001/.well-known/appspecific/com.chrome.devtools.json (Référent: None) [in X:\MediaManagerSuite\app\__init__.py:79]
2025-06-14 17:32:59,763 INFO: MediaManagerSuite startup in production mode [in X:\MediaManagerSuite\app\__init__.py:35]
2025-06-14 17:33:00,177 INFO: Blueprint 'plex_editor' enregistré avec succès. [in X:\MediaManagerSuite\app\__init__.py:46]
2025-06-14 17:33:00,326 INFO: Blueprint 'seedbox_ui' enregistré avec succès. [in X:\MediaManagerSuite\app\__init__.py:51]
2025-06-14 17:33:00,330 INFO: Blueprint 'config_ui' enregistré avec succès. [in X:\MediaManagerSuite\app\__init__.py:59]
2025-06-14 17:33:00,330 INFO: Application MediaManagerSuite créée et configurée. [in X:\MediaManagerSuite\app\__init__.py:88]
2025-06-14 17:33:00,331 INFO: Démarrage MediaManagerSuite. Debug: False, Reloader: False (pour test) [in X:\MediaManagerSuite\run.py:25]
2025-06-14 17:33:06,334 WARNING: Erreur 404 - Page non trouvée: http://localhost:5001/.well-known/appspecific/com.chrome.devtools.json (Référent: None) [in X:\MediaManagerSuite\app\__init__.py:79]
2025-06-14 17:33:09,399 INFO: Utilisateur 'User' (ID: 36440265) sélectionné. [in X:\MediaManagerSuite\app\plex_editor\routes.py:75]
2025-06-14 17:33:09,708 WARNING: Erreur 404 - Page non trouvée: http://localhost:5001/.well-known/appspecific/com.chrome.devtools.json (Référent: None) [in X:\MediaManagerSuite\app\__init__.py:79]
2025-06-14 17:33:11,378 INFO: show_library: Arguments finaux pour API search(): {'sort': 'addedAt:desc'} [in X:\MediaManagerSuite\app\plex_editor\routes.py:223]
2025-06-14 17:33:11,773 INFO: Emprunt d'identité réussi pour 'User'. [in X:\MediaManagerSuite\app\plex_editor\routes.py:245]
2025-06-14 17:33:12,033 INFO: Exécution de library.search sur 'Films' pour utilisateur 'User' avec args: {'sort': 'addedAt:desc'} [in X:\MediaManagerSuite\app\plex_editor\routes.py:270]
2025-06-14 17:33:12,216 INFO: DEBUG_TAG: La valeur de RADARR_TAG_ON_ARCHIVE est 'os.getenv('RADARR_TAG_ON_ARCHIVE', 'vu')' [in X:\MediaManagerSuite\app\plex_editor\routes.py:336]
2025-06-14 17:33:13,985 WARNING: Erreur 404 - Page non trouvée: http://localhost:5001/.well-known/appspecific/com.chrome.devtools.json (Référent: None) [in X:\MediaManagerSuite\app\__init__.py:79]
2025-06-14 17:33:23,100 INFO: Filtre date API: ajout_recent_jours = 100d [in X:\MediaManagerSuite\app\plex_editor\routes.py:180]
2025-06-14 17:33:23,102 INFO: show_library: Arguments finaux pour API search(): {'unwatched': False, 'addedAt>>=': '100d', 'sort': 'addedAt:desc'} [in X:\MediaManagerSuite\app\plex_editor\routes.py:223]
2025-06-14 17:33:23,507 INFO: Emprunt d'identité réussi pour 'User'. [in X:\MediaManagerSuite\app\plex_editor\routes.py:245]
2025-06-14 17:33:23,513 INFO: Exécution de library.search sur 'Films' pour utilisateur 'User' avec args: {'unwatched': False, 'addedAt>>=': '100d', 'sort': 'addedAt:desc'} [in X:\MediaManagerSuite\app\plex_editor\routes.py:270]
2025-06-14 17:33:23,533 INFO: DEBUG_TAG: La valeur de RADARR_TAG_ON_ARCHIVE est 'os.getenv('RADARR_TAG_ON_ARCHIVE', 'vu')' [in X:\MediaManagerSuite\app\plex_editor\routes.py:336]
2025-06-14 17:33:23,882 WARNING: Erreur 404 - Page non trouvée: http://localhost:5001/.well-known/appspecific/com.chrome.devtools.json (Référent: None) [in X:\MediaManagerSuite\app\__init__.py:79]
2025-06-14 17:40:23,285 INFO: MediaManagerSuite startup in production mode [in X:\MediaManagerSuite\app\__init__.py:35]
2025-06-14 17:40:23,701 INFO: Blueprint 'plex_editor' enregistré avec succès. [in X:\MediaManagerSuite\app\__init__.py:46]
2025-06-14 17:40:23,854 INFO: Blueprint 'seedbox_ui' enregistré avec succès. [in X:\MediaManagerSuite\app\__init__.py:51]
2025-06-14 17:40:23,858 INFO: Blueprint 'config_ui' enregistré avec succès. [in X:\MediaManagerSuite\app\__init__.py:59]
2025-06-14 17:40:23,859 INFO: Application MediaManagerSuite créée et configurée. [in X:\MediaManagerSuite\app\__init__.py:88]
2025-06-14 17:40:23,859 INFO: Démarrage MediaManagerSuite. Debug: False, Reloader: False (pour test) [in X:\MediaManagerSuite\run.py:25]
2025-06-14 17:40:35,608 WARNING: Erreur 404 - Page non trouvée: http://localhost:5001/.well-known/appspecific/com.chrome.devtools.json (Référent: None) [in X:\MediaManagerSuite\app\__init__.py:79]
2025-06-14 17:40:38,316 INFO: Utilisateur 'User' (ID: 36440265) sélectionné. [in X:\MediaManagerSuite\app\plex_editor\routes.py:75]
2025-06-14 17:40:38,631 WARNING: Erreur 404 - Page non trouvée: http://localhost:5001/.well-known/appspecific/com.chrome.devtools.json (Référent: None) [in X:\MediaManagerSuite\app\__init__.py:79]
2025-06-14 17:40:39,855 INFO: show_library: Arguments finaux pour API search(): {'sort': 'addedAt:desc'} [in X:\MediaManagerSuite\app\plex_editor\routes.py:223]
2025-06-14 17:40:40,271 INFO: Emprunt d'identité réussi pour 'User'. [in X:\MediaManagerSuite\app\plex_editor\routes.py:245]
2025-06-14 17:40:40,611 INFO: Exécution de library.search sur 'Films' pour utilisateur 'User' avec args: {'sort': 'addedAt:desc'} [in X:\MediaManagerSuite\app\plex_editor\routes.py:270]
2025-06-14 17:40:40,819 INFO: DEBUG_TAG: La valeur de RADARR_TAG_ON_ARCHIVE est 'vu' [in X:\MediaManagerSuite\app\plex_editor\routes.py:336]
2025-06-14 17:40:42,628 WARNING: Erreur 404 - Page non trouvée: http://localhost:5001/.well-known/appspecific/com.chrome.devtools.json (Référent: None) [in X:\MediaManagerSuite\app\__init__.py:79]
2025-06-14 17:40:51,663 INFO: Filtre date API: ajout_recent_jours = 100d [in X:\MediaManagerSuite\app\plex_editor\routes.py:180]
2025-06-14 17:40:51,665 INFO: show_library: Arguments finaux pour API search(): {'unwatched': False, 'addedAt>>=': '100d', 'sort': 'addedAt:desc'} [in X:\MediaManagerSuite\app\plex_editor\routes.py:223]
2025-06-14 17:40:52,069 INFO: Emprunt d'identité réussi pour 'User'. [in X:\MediaManagerSuite\app\plex_editor\routes.py:245]
2025-06-14 17:40:52,076 INFO: Exécution de library.search sur 'Films' pour utilisateur 'User' avec args: {'unwatched': False, 'addedAt>>=': '100d', 'sort': 'addedAt:desc'} [in X:\MediaManagerSuite\app\plex_editor\routes.py:270]
2025-06-14 17:40:52,095 INFO: DEBUG_TAG: La valeur de RADARR_TAG_ON_ARCHIVE est 'vu' [in X:\MediaManagerSuite\app\plex_editor\routes.py:336]
2025-06-14 17:40:52,446 WARNING: Erreur 404 - Page non trouvée: http://localhost:5001/.well-known/appspecific/com.chrome.devtools.json (Référent: None) [in X:\MediaManagerSuite\app\__init__.py:79]
2025-06-14 17:51:36,072 INFO: MediaManagerSuite startup in production mode [in X:\MediaManagerSuite\app\__init__.py:35]
2025-06-14 17:51:36,485 INFO: Blueprint 'plex_editor' enregistré avec succès. [in X:\MediaManagerSuite\app\__init__.py:46]
2025-06-14 17:51:36,636 INFO: Blueprint 'seedbox_ui' enregistré avec succès. [in X:\MediaManagerSuite\app\__init__.py:51]
2025-06-14 17:51:36,642 INFO: Blueprint 'config_ui' enregistré avec succès. [in X:\MediaManagerSuite\app\__init__.py:59]
2025-06-14 17:51:36,642 INFO: Application MediaManagerSuite créée et configurée. [in X:\MediaManagerSuite\app\__init__.py:88]
2025-06-14 17:51:36,642 INFO: Démarrage MediaManagerSuite. Debug: False, Reloader: False (pour test) [in X:\MediaManagerSuite\run.py:25]
2025-06-14 17:51:40,544 WARNING: Erreur 404 - Page non trouvée: http://localhost:5001/.well-known/appspecific/com.chrome.devtools.json (Référent: None) [in X:\MediaManagerSuite\app\__init__.py:79]
2025-06-14 17:51:43,607 INFO: Utilisateur 'User' (ID: 36440265) sélectionné. [in X:\MediaManagerSuite\app\plex_editor\routes.py:75]
2025-06-14 17:51:43,914 WARNING: Erreur 404 - Page non trouvée: http://localhost:5001/.well-known/appspecific/com.chrome.devtools.json (Référent: None) [in X:\MediaManagerSuite\app\__init__.py:79]
2025-06-14 17:51:45,163 INFO: show_library: Arguments finaux pour API search(): {'sort': 'addedAt:desc'} [in X:\MediaManagerSuite\app\plex_editor\routes.py:223]
2025-06-14 17:51:45,542 INFO: Emprunt d'identité réussi pour 'User'. [in X:\MediaManagerSuite\app\plex_editor\routes.py:245]
2025-06-14 17:51:45,803 INFO: Exécution de library.search sur 'Films' pour utilisateur 'User' avec args: {'sort': 'addedAt:desc'} [in X:\MediaManagerSuite\app\plex_editor\routes.py:270]
2025-06-14 17:51:45,994 INFO: DEBUG_TAG: La valeur de RADARR_TAG_ON_ARCHIVE est 'vu' [in X:\MediaManagerSuite\app\plex_editor\routes.py:336]
2025-06-14 17:51:47,748 WARNING: Erreur 404 - Page non trouvée: http://localhost:5001/.well-known/appspecific/com.chrome.devtools.json (Référent: None) [in X:\MediaManagerSuite\app\__init__.py:79]
2025-06-14 17:52:23,357 INFO: Filtre date API: ajout_recent_jours = 100d [in X:\MediaManagerSuite\app\plex_editor\routes.py:180]
2025-06-14 17:52:23,360 INFO: show_library: Arguments finaux pour API search(): {'unwatched': False, 'addedAt>>=': '100d', 'sort': 'addedAt:desc'} [in X:\MediaManagerSuite\app\plex_editor\routes.py:223]
